### PR TITLE
Add Nominator Minimum Stake Section

### PR DESCRIPTION
**Pull request by rskntroot**
Add a section regarding minimum required DOT for staking.

### CODE REVIEW

File: docs/learn-nominator.md [hunk 1]
```diff
@@ -46,6 +46,12 @@ nomination. However, the election algorithm attempts to minimize this situation,
 occur often, so you should almost always see only a single active nomination per era. See the
 [section on Phragmén optimization](learn-phragmen#optimizations) for more details.
 
+### Required Minimum Stake
+
+To be considered a Nominator there is minimum amount of DOT you must stake.
+
+As of Februrary 2021, this minimum is 104 DOT.
+
```
Comment (Swader) [inline docs/learn-nominator.md]:
```suggestion
### Required Minimum Stake

Due to the overwhelming demand and the way the algorithm optimizes the nominator set,
there is a minimum number of DOT you must nominate with in order to be effective.

This number fluctuates day by day, and the most recent and up to date version can be
found out using [this community-developed tool](https://polkaview.network/dot) or
through [these scripts](https://github.com/w3f/validator-stats) which you can run on your machine.

```

**Review by Swader — CHANGES_REQUESTED** [review]
We don't want to be specific with any numbers because they change too frequently.

**Comment by rskntroot** [comment]
> We don't want to be specific with any numbers because they change too frequently.

Was not aware of the tool, appreciate the changes.

**Comment by Swader** [comment]
@laboon I am linking here to a community made tool. It's very practical and in my opinion safe to link to, but wanted to know if you had a different opinion.

**Comment by laboon** [comment]
I think if it's OK if we note that it is a community-generated tools. But I want to change the wording on this.

File: docs/learn-nominator.md [hunk 2]
```diff
@@ -46,6 +46,15 @@ nomination. However, the election algorithm attempts to minimize this situation,
 occur often, so you should almost always see only a single active nomination per era. See the
 [section on Phragmén optimization](learn-phragmen#optimizations) for more details.
 
+### Required Minimum Stake
+
+Due to the overwhelming demand and the way the algorithm optimizes the nominator set,
```
Comment (laboon) [inline docs/learn-nominator.md]:
```suggestion
Due to the way the [Phragmen algorithm](learn-phragmen) generates the solution set,
and due to the fact that the solution set must fit in a single block, 
```

File: docs/learn-nominator.md [hunk 3]
```diff
@@ -46,6 +46,15 @@ nomination. However, the election algorithm attempts to minimize this situation,
...
+there is a minimum number of DOT you must nominate with in order to be effective.
```
Comment (laboon) [inline docs/learn-nominator.md]:
```suggestion
there is a minimum number of DOT you must nominate with in order to receive staking rewards.
```

File: docs/learn-nominator.md [hunk 4]
```diff
@@ -46,6 +46,15 @@ nomination. However, the election algorithm attempts to minimize this situation,
...
+
+This number fluctuates day by day, and the most recent and up to date version can be
```
Comment (laboon) [inline docs/learn-nominator.md]:
```suggestion
This number fluctuates every era, and the most recent and up to date version can be
```

File: docs/learn-nominator.md [hunk 5]
```diff
@@ -46,6 +46,15 @@ nomination. However, the election algorithm attempts to minimize this situation,
...
+found out using [this community-developed tool](https://polkaview.network/dot) or
+through [these scripts](https://github.com/w3f/validator-stats) which you can run on your machine.
+
```
Comment (laboon) [inline docs/learn-nominator.md]:
```suggestion
See the [Election Solution Set](https://wiki.polkadot.network/docs/en/learn-nominator#the-election-solution-set) section on the Nominator page for more details.
```

**Review by laboon — CHANGES_REQUESTED** [review]

**Comment by rskntroot** [comment]
@laboon - thanks for the clarification, the changes read well.

**Comment by Swader** [comment]
Nice, thanks

**Pull request merged**